packages/nodes-base/test/nodes/If/If.node.test.ts

```diff
@@ -0,0 +1,59 @@
+import { INodeType } from 'n8n-workflow';
+import * as Helpers from '../Helpers';
+import { WorkflowTestData } from '../types';
+import { readFileSync } from 'fs';
+
+import { ManualTrigger } from '../../../nodes/ManualTrigger/ManualTrigger.node';
+import { Set } from '../../../nodes/Set/Set.node';
+import { If } from '../../../nodes/If/If.node';
+import { NoOp } from '../../../nodes/NoOp/NoOp.node';
+import { Code } from '../../../nodes/Code/Code.node';
+
+import { executeWorkflow } from '../ExecuteWorkflow';
+
+describe('Execute If Node', () => {
+	const tests: Array<WorkflowTestData> = [
+		{
+			description: 'should execute IF node true/false boolean',
+			input: {
+				workflowData: JSON.parse(readFileSync('test/nodes/If/workflow.json', 'utf-8')),
```

Comment:
we could add some helper that auto detects path here

Reply:
Have implemented very basic helper but we still need to be able to load multiple workflow jsons per unit test if we need to, so some kind of filename or path would be required.